src/store/record.js

```diff
@@ -298,6 +300,39 @@ let recordStore = {
             let organisations = await client.executeQuery(recordOrganisationsQuery);
             commit('updateOrganisationsLinks', organisations.fairsharingRecord.organisationLinks);
         },
+        async updateAdditionalInformation({ state, commit}, options){
+            commit("resetMessage", "additionalInformation");
+            let access_points = state.sections.additionalInformation.data.access_points;
+            /*
+                What this does:
+                1. Get metadata (new GraphQL query)
+                2. Modify with access_points,
+                3. Post again (restClient.updateRecord).
+             */
+            metadataQuery.queryParam = {
+                id: options.id
+            }
+            let serverMetadata = await client.executeQuery(metadataQuery);
+            let metadata = serverMetadata['fairsharingRecord']['metadata'];
+            metadata['access_points'] = access_points;
+            // define record
+            const record = {
+                record: { metadata: metadata },
+                token: options.token,
+                id: options.id
+            };
+            let response = await restClient.updateRecord(record);
+            if (response.error) {
+                commit("setSectionError", {
+                    section: "additionalInformation",
+                    value: response.error
+                });
+                return response.error;
+            }
+            else {
+                commit("setMessage", {target: "additionalInformation", value: "Record successfully updated!"});
```

Comment:
We need a point of discussion here: updating the other sections that rely on metadata to avoid overwritting when saving.